RestAPI_Automation/Requests/APIHelper.cs

```diff
@@ -0,0 +1,54 @@
+﻿using Newtonsoft.Json;
+using RestSharp;
+
+namespace RestAPI_Automation.Requests;
+
+public class APIHelper<T>
+{
+    public string baseURL = "https://reqres.in/ ";
+
+    // concatenating base url and endpoint
+    public string SetUrl(string endpoint)
+    {
+        string url = Path.Combine(baseURL, endpoint);
+        return url;
+    }
+
+    // method returns RestClient value
+    public RestClient RestClient()
+    {
+        var client = new RestClient(baseURL);
+        return client;
+    }
+
+    // POST request ulr, header and parameters
+    public RestRequest CreatePostRequest(string payload, string endpoint)
+    {
+        var restRequest = new RestRequest(endpoint, Method.Post);
+        restRequest.AddHeader("Accept", "application/json");
```

Comment:
1. "application/json" can be replaces with MediaTypeNames.Application.Json
2. "Accept" can be replaced with HttpRequestHeader.Accept

Reply:
corrected